#!/usr/bin/env python
PACKAGE = "ainstein_radar_filters"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add( "min_range", double_t, 0, "Minimum target range, in meters", 0.5, 0.0, 100.0 )
gen.add( "max_range", double_t, 0, "Maximum target range, in meters", 10.0, 0.0, 100.0 )

exit(gen.generate(PACKAGE, "ainstein_radar_filters", "RangeFilter"))
